src/Sylius/Bundle/CoreBundle/EventListener/OrderTaxationListener.php

```diff
@@ -50,6 +50,6 @@ public function applyTaxes(GenericEvent $event)
             );
         }
 
-        $this->orderTaxesApplicator->apply($order);
+        $this->orderTaxesProcessor->applyTaxes($order);
```

Comment:
processTaxes? Having 2 interfaces related to taxes that have apply method can be confusing.